src/GameController/GCExtendedGamepadSnapshot.cs

```diff
@@ -47,6 +47,11 @@ public struct GCExtendedGamepadSnapShotDataV100 {
 		public float /* float_t = float */ LeftTrigger;
 		public float /* float_t = float */ RightTrigger;
 
+		[TV (12, 0), Mac (10, 14, onlyOn64: true), iOS (12, 0)]
```

Comment:
Shouldn't the version in the attributes be 12.1 and not 12.0 (and 10.14.1 for macOS)?

Reply:
Mmh indeed it looks like intro isn't happy about those selectors on iOS 12.0. Sometimes it works though (private APIs made public).